src/cpp/Root.cpp

```diff
@@ -197,11 +194,9 @@ bool Root::get_next_client(std::shared_ptr<ProxyClient>& next_client)
     return rv;
 }
 
-bool Root::load_config_file(const std::string& path)
+bool Root::load_config_file(const std::string& file)
```

Comment:
You can not put the path? Maybe file_path?

Reply:
Modified!